samples/unit-tests/axis/ordinal/demo.js

```diff
@@ -767,6 +775,9 @@ QUnit.test('Moving annotations on ordinal axis, #18459', assert => {
 
     const chart = Highcharts.stockChart('container', {
         series: [{
+            dataGrouping: {
+                forced: true
+            },
```

Comment:
Please, keep the original test and update series for your scenario. I don't know if the `Annotation dragged on ordinal axis charts should follow mouse pointer` assertion will be fine if we force grouping.